### PR TITLE
Dtd 2216

### PR DESCRIPTION
**Pull request by rogerball**

### CODE REVIEW

File: src/test/resources/features/ifs/MultipeDebtItems EdgeCases.feature [hunk 1]
```diff
@@ -252,3 +225,19 @@ Feature: Multiple Debt Items - Edge Cases
       | 2020-10-16 | 2020-12-31 | 76           | 2.6          | 28                      | 402159             |
       | 2021-01-01 | 2021-02-22 | 53           | 2.6          | 28                      | 401510             |
 
+  @DTD-2216 @runMe
```
Comment (thadeusssewanyana) [inline src/test/resources/features/ifs/MultipeDebtItems EdgeCases.feature]: tag

**Review by thadeusssewanyana — APPROVED** [review]
